src/components/IntegrationsDropdown.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { Dropdown, DropdownItem, DropdownList, MenuToggle } from '@patternfly/react-core';
+import { AsyncComponent } from '@redhat-cloud-services/frontend-components';
+import AddSourceWizard from './addSourceWizard';
+import { CLOUD_VENDOR, COMMUNICATIONS, REDHAT_VENDOR, REPORTING, WEBHOOKS } from '../utilities/constants';
+
+const dropdownItems = [
+  {
+    title: 'Communications',
+    description: 'Google Chat, Microsoft Office Teams, Slack',
+    value: COMMUNICATIONS,
+  },
+  {
+    title: 'Reporting & Automation',
+    description: 'Event-Driven Ansible, ServiceNow, Splunk',
+    value: REPORTING,
+  },
+  {
+    title: 'Webhooks',
+    description: '',
+    value: WEBHOOKS,
+  },
+  {
+    title: 'Cloud',
+    description: 'Amazon Web Services, Google Cloud Platform, Microsoft Azure, Oracle Cloud Platform',
+    value: CLOUD_VENDOR,
+  },
+  {
+    title: 'Red Hat',
+    description: 'Red Hat OpenShift Container Platform, Red Hat Satellite, Red Hat Ansible Automation Platform',
+    value: REDHAT_VENDOR,
+  },
+];
+
+const IntegrationsDropdown = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isIntegrationsWizardOpen, setIsIntegrationsWizardOpen] = useState(false);
+  const [isSourcesWizardOpen, setIsSourcesWizardOpen] = useState(false);
+  const [selectedIntegration, setSelectedIntegration] = useState(null);
+
+  const handleSelect = (_event, value) => {
+    setIsOpen(false);
+    console.log('Selected:', value);
+    setSelectedIntegration(value);
+    if (value === CLOUD_VENDOR || value === REDHAT_VENDOR) {
+      setIsSourcesWizardOpen(true);
+    } else if (value === COMMUNICATIONS || value === REPORTING || value === WEBHOOKS) {
+      setIsIntegrationsWizardOpen(true);
+    }
+  };
+
+  return (
+    <div id="integrations-dropdown">
+      <AsyncComponent
+        appName="notifications"
+        module="./IntegrationsWizard"
+        isOpen={isIntegrationsWizardOpen}
+        category={selectedIntegration}
+        closeModal={() => setIsIntegrationsWizardOpen(false)}
+        fallback={<div id="fallback-modal" />}
+      />
+      <AddSourceWizard
+        isOpen={isSourcesWizardOpen}
+        onClose={() => setIsSourcesWizardOpen(false)}
+        activeCategory={selectedIntegration}
+      />
```

Comment:
You should wrap the sources wizard and mount it only if correct source is selected
```suggestion
      {[REDHAT_VENDOR, CLOUD_VENDOR].includes(selectedIntegration) && <AddSourceWizard
        isOpen={isSourcesWizardOpen}
        onClose={() => {
          // let's se the selected integration to null so sources wizard unmounts
          setSelectedIntegration(null);
          setIsSourcesWizardOpen(false);
        }}
        activeCategory={selectedIntegration}
      />}
```